src/transaction.ts

```diff
@@ -59,6 +59,21 @@ export interface RequestOptions {
   jsonOptions?: JSONOptions;
   gaxOptions?: CallOptions;
   maxResumeRetries?: number;
+  /**
+   * An object where column names as keys and custom objects as corresponding
+   * values for deserialization. It's specifically useful for data types like
+   * protobuf where deserialization logic is on user-specific code. When provided,
+   * the custom object enables deserialization of backend-received column data.
+   * If not provided, data remains serialized as buffer for Proto Messages and
+   * integer for Proto Enums.
+   *
+   * @example
+   * To obtain Proto Messages and Proto Enums as JSON objects, the customer needs
+   * to supply a custom parameter. This parameter should include the protobufjs-cli
```

Comment:
Also this comment and also the one above could really use a concrete example of what we expect users to pass in. The current description is very abstract and will just be confusing for users who are not using proto columns.

I would suggest that we:
1. Change the wording to 'this is only needed for proto columns' (as opposed to 'it is specifically useful for data types like protobu', as that sounds like it could also be used for other data types)
2. Add a concrete example of what people should pass in based on our commonly used examples that should be recognizable for our users.

```suggestion
   * To obtain Proto Messages and Proto Enums as JSON objects, you must
   * supply a additional metadata. This metadata should include the protobufjs-cli
```

Reply:
Thanks Knut for the suggestion. I have added an example. Once the PR is merged. I will also add a link to the generated file.